### PR TITLE
Update "undici" package dependency of vercel/node pkg

### PR DESCRIPTION
**Pull request by martinez-hugo**
This refer to this threat : https://github.com/advisories/GHSA-wqq4-5wpv-mx2g

### CODE REVIEW

File: .changeset/pink-otters-teach.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@vercel/node': major
```
Comment (TooTallNate) [inline .changeset/pink-otters-teach.md]:
```suggestion
'@vercel/node': patch
```

**Review by Ethan-Arrowood — APPROVED** [review]

**Comment by martinez-hugo** [comment]
Thanks @Ethan-Arrowood 
Do you think that you can approve on Vercel this request ?

**Review by cb1kenobi — APPROVED** [review]

**Comment by martinez-hugo** [comment]
Thanks @cb1kenobi ! There are a problem I think about changeset, but I don't know about what.

Logs said `error: pathspec 'update-prod-pkg' did not match any file(s) known to git`when it run `git checkout update-prod-pkg`

**Comment by cb1kenobi** [comment]
Running tests here: https://github.com/vercel/vercel/pull/10763

**Comment by martinez-hugo** [comment]
> Running tests here: #10763

I think now it’s deployment on vercel that’s blocking checks
Do you have visibility on deployment logs ?

**Comment by martinez-hugo** [comment]
Nice it works for deployment @cb1kenobi @Ethan-Arrowood @TooTallNate  so I think that Tests will pass now if we relaunch it

**Comment by cb1kenobi** [comment]
@martinez-hugo we are experiencing degraded performance and will re-run the tests as soon as the system returns back to normal.

**Comment by martinez-hugo** [comment]
@cb1kenobi @TooTallNate @Ethan-Arrowood I just sync branch with main branch

**Comment by martinez-hugo** [comment]
I don’t understand how these errors can exists. Because I just update vercel/node, not vercel core

**Comment by Ethan-Arrowood** [comment]
It's not you, its us lol! Our dev server implementation is broken by the recent Undici update. This is okay, we were using fetch someplace we shouldn't have been. This PR: https://github.com/vercel/vercel/pull/10767 fixes it but will take a moment to land. I'd like to add you as a co-commit-author on the update undici commit. What is your preferred git user name and email address (something like `Co-authored-by: Ethan Arrowood <ethan.arrowood@vercel.com>`)

**Comment by martinez-hugo** [comment]
> It's not you, its us lol! Our dev server implementation is broken by the recent Undici update. This is okay, we were using fetch someplace we shouldn't have been. This PR: #10767 fixes it but will take a moment to land. I'd like to add you as a co-commit-author on the update undici commit. What is your preferred git user name and email address (something like `Co-authored-by: Ethan Arrowood <ethan.arrowood@vercel.com>`)

Thanks for all Ethan. 
Can I help on anything on this fix ?
This is my good informations : Hugo Martinez <contact@hmartinez.fr>

**Comment by Ethan-Arrowood** [comment]
Thank you again for bringing this to our attention! We've released version 32.5.3 which contains the undici dependency upgrade as well as the necessary fix for the update. 🚀

**Pull request closed without merging**